src/components/Article.jsx

```diff
@@ -1 +1,31 @@
-// export const Article = () => ();
+import React from 'react';
+
+export const Article = (props) => {
+  const {
+    title,
+    date,
+    text,
+  } = props;
+
+  return (
+    <>
+      <h2 className="title" data-cy="title">
+        {title}
+      </h2>
+
+      <p className="subtitle">
+        <time dateTime={date.toISOString().slice(0, 10)} data-cy="date">
+          {date.toLocaleDateString('en-US', {
+            year: 'numeric',
+            month: 'long',
+            day: 'numeric',
+          })}
+        </time>
+      </p>
+
+      <p data-cy="text">
+        {text}
+      </p>
+    </>
+  );
+};
```

Comment:
The code is all good bur your demo link is not working.

Reply:
Try now!